### ISSUE TITLE
URL field text invisible when text is entered

### ISSUE THREAD

**Issue by Huskion** [MerlinVR/USharpVideo#51]
Hi, the UI appears correctly, but once anything is entered into the "Text" field, the placeholder ("Enter Video URL...") disappears as expected, yet the actual typed text isn't showing. The entered URL is invisible, but a user knows that text has been entered because the typing cursor moves, and pressing the play button on a valid URL works. So the text is there but you can't see it.

This happens on multiple projects of mine - Reimporting the player, deleting it completely and importing again from scratch hasn't helped. SDK is all up to date.

Wondering if there is a TextMeshPro setting conflict for projects where the VRC Billiards Community Edition is also present. Any ideas?

**Comment by MerlinVR** [comment]
USharpVideo doesn't use TMP, Unity can bug out with text fields sometimes. Make sure you are importing from the unitypackage on the releases page.

**Comment by Huskion** [comment]
Yep, definitely am doing that. Even redownloaded it to ensure that it's not corrupted or anything.

**Comment by MerlinVR** [comment]
Not sure then, never have run into that or heard of it being an issue with clean imports. You might want to try importing in a clean project then, something may be breaking it in the projects you've imported it into.

**Comment by Huskion** [comment]
Hello, after some troubleshooting, it appears that disabling/hiding the "SyncModeToggle" element (preventing users from toggling Video/Stream mode) causes the text to disappear. What is the correlation between these two UI elements that requires the toggle to be on so that text can be seen in the URL bar?
